src/telliot_feeds/reporters/tellorflex.py

```diff
@@ -249,6 +248,8 @@ async def fetch_datafeed(self) -> Optional[DataFeed[Any]]:
         """Fetches datafeed suggestion plus the reward amount from autopay if query tag isn't selected
         if query tag is selected fetches the rewards, if any, for that query tag"""
         if self.datafeed:
+            if self.datafeed.query.query_id not in CATALOG_QUERY_IDS:
+                CATALOG_QUERY_IDS[self.datafeed.query.query_id] = self.datafeed.query.descriptor
```

Comment:
I think we want `self.datafeed.query.query_data`, no?

Reply:
the reason I used descriptor is because the dictionary is ```{query_id: tag}``` and since there won't be a tag for newly built query_ids, this would have more info; at least thats how I thought about it

Reply:
That makes sense